File: aulas/aula_1/teste lista.py
lista_nome = []

lista_nome.append("Gabriel")

print(lista_nome)

if len(lista_nome) != 0:
    nome = input("Digite um nome: ")
    if lista_nome.__contains__(nome):
        lista_nome.remove("nome")
    else:
        print("Nome não está na lista")

    print(lista_nome)
else:
    print("Lista vazia")

File: aulas/aula_1/teste_lista.java
import java.util.ArrayList;
import java.util.Scanner;

public class teste_lista
{
    public static void main(String[] args)
    {
        ArrayList<String> listaNome = new ArrayList<>();

        listaNome.add("Gabriel");

        System.out.println(listaNome);

        if(!listaNome.isEmpty())
        {
            Scanner teclado = new Scanner(System.in);

            System.out.print("Digite um nome: ");
            String nome = teclado.nextLine();

            if(listaNome.contains(nome))
            {
                listaNome.remove(nome);
            }
            else
            {
                System.out.println("Nome não está na lista");
            }

            System.out.println(listaNome);
            teclado.close();
        }
        else
        {
            System.out.println("Lista vazia");
        }
    }
}